"cookie" module has no default export

## Describe the bug
This is more of a convenience issue, than bug.

Performing a build with TypeScript 3.9.5 with `allowSyntheticDefaultImports` disabled displays:
> ERROR in node_modules/msw/lib/types/context/cookie.d.ts:1:8 - error TS1192: Module '"<path>/node_modules/@types/cookie/index"' has no default export.

Now the issue is because my TypeScript config is strict, and I can modify it to either ignore libraries or allow the synthetic import. However, it might be better to update areas of the code that are doing this:

```typescript
import cookieUtils from 'cookie'
```
To

```typescript
import { parse } from 'cookie'
```
Since the [library definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cookie/index.d.ts) accurately reflect no default export on the [cookie](https://github.com/jshttp/cookie/blob/master/index.js) package.

I also could have all this wrong... I don't claim to be an expert in any of this

## Environment

- `msw: 0.20.1`
- `nodejs: 14.2.0`
- `npm: 6.14.5`

> Please also provide your browser version.

## To Reproduce
Steps to reproduce the behavior:
1. Configure TypeScript compilation without `allowSyntheticDefaultImports`
2. Import `import { setupWorker, rest } from 'msw';` somewhere in code
3. TypeScript compilation issue


## Expected behavior
No compilation errors

## Screenshots
If applicable, add screenshots to help explain your problem.

--- a/src/context/cookie.test.ts
+++ b/src/context/cookie.test.ts
@@ -1,4 +1,4 @@
-import cookieUtils from 'cookie'
+import * as cookieUtils from 'cookie'
 import { cookie } from './cookie'
 import { response } from '../response'
 

--- a/src/context/cookie.ts
+++ b/src/context/cookie.ts
@@ -1,4 +1,4 @@
-import cookieUtils, { CookieSerializeOptions } from 'cookie'
+import * as cookieUtils from 'cookie'
 import { ResponseTransformer } from '../response'
 
 /**
@@ -9,14 +9,16 @@ import { ResponseTransformer } from '../response'
 export const cookie = (
   name: string,
   value: string,
-  options?: CookieSerializeOptions,
+  options?: cookieUtils.CookieSerializeOptions,
 ): ResponseTransformer => {
   return (res) => {
     const serializedCookie = cookieUtils.serialize(name, value, options)
     res.headers.set('Set-Cookie', serializedCookie)
+
     if (typeof document !== 'undefined') {
       document.cookie = serializedCookie
     }
+
     return res
   }
 }

--- a/src/context/status.ts
+++ b/src/context/status.ts
@@ -1,4 +1,4 @@
-import statuses from 'statuses/codes.json'
+import * as statuses from 'statuses/codes.json'
 import { ResponseTransformer } from '../response'
 
 export const status = (

--- a/src/node/setupServer.ts
+++ b/src/node/setupServer.ts
@@ -1,4 +1,4 @@
-import cookieUtils from 'cookie'
+import * as cookieUtils from 'cookie'
 import { Headers, flattenHeadersObject } from 'headers-utils'
 import {
   RequestInterceptor,

--- a/src/utils/request/getRequestCookies.ts
+++ b/src/utils/request/getRequestCookies.ts
@@ -1,4 +1,4 @@
-import cookieUtils from 'cookie'
+import * as cookieUtils from 'cookie'
 import { MockedRequest } from '../../handlers/requestHandler'
 
 function getAllCookies() {

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -5,6 +5,7 @@
     "target": "es6",
     "module": "ESNext",
     "moduleResolution": "node",
+    "allowSyntheticDefaultImports": false,
     "esModuleInterop": true,
     "resolveJsonModule": true,
     "declaration": true,

